src/pages/index.js

```diff
@@ -1,12 +1,16 @@
 import React from "react"
 import { graphql, Link } from "gatsby"
 
-import Layout from "../components/layout"
+import "bootstrap/dist/css/bootstrap.min.css"
+
+// import Layout from "../components/layout"
 import Image from "../components/image"
 import SEO from "../components/seo"
+import Navbar from "../components/navbar"
 
 const IndexPage = ({ data }) => (
-  <Layout>
+  <div>
```

Comment:
Not sure how the development is going, but a `Layout` component might be useful, given the way that gatsby handles pages. Probably needs some discussion.

Reply:
We don't have a Layout component since the Image department is still working on other pages, so we don't know exactly what the component would have. We do have a mockup for the Navbar, which is why it's included here.

Reply:
Right, but maybe there can be a `Layout` component with the navbar, for example, so it is included in every page 😄 

It's just a suggestion, and I'm not sure if it's the best course of action for the future, just throwing the idea out there